applications/FluidDynamicsApplication/custom_elements/fluid_element.cpp

```diff
@@ -553,6 +553,20 @@ array_1d<double, 3> FluidElement<TElementData>::GetAtCoordinate(
     return result;
 }
 
+template <class TElementData>
+BoundedMatrix<double, TElementData::Dim, TElementData::Dim> FluidElement<TElementData>::GetAtCoordinate(
+    const typename TElementData::NodalTensorData &rValues,
+    const typename TElementData::ShapeFunctionsType &rN) const
+{
+    BoundedMatrix<double,Dim,Dim> result = ZeroMatrix(Dim,Dim);
```

Comment:
This is only valid for diagonal matrices, isn't it?

Reply:
No, you can define the permeability as a dense matrix as well.

Reply:
oh.. I see